docs/assets/funfacts.txt

```diff
@@ -1,6 +1,7 @@
 RIP Technoblade\nHe will be missed
 RIP Technoblade\nHe will be missed
 RIP Technoblade\nHe will be missed
+RIP Light_Exists\nHe will be missed
```

Comment:
No results found. Empathy banana is here for you.

Reply:
> <span>#</span>1 potato farmer
i knew him a little bit but not too well 
he comitted suicide to my knowledge 
i stumbled upon melonking's bio and decided to add it